doc/command_ref.rst

```diff
@@ -1760,12 +1760,14 @@ Upgrade-Minimal Command
 
 ``dnf [options] upgrade-minimal``
     Updates each package to the latest available version that provides a bugfix, enhancement
-    or a fix for a security issue (security).
+    or a fix for a security issue (security). Newer updates not mentioned in any advisories
+    are ignored.
 
 ``dnf [options] upgrade-minimal <package-spec>...``
     Updates each specified package to the latest available version that provides
-    a bugfix, enhancement or a fix for security issue (security). Updates
-    dependencies as necessary.
+    a bugfix, enhancement or a fix for security issue (security). Newer updates
+    not mentioned in any advisories are ignored. Dependencies are updated
+    as needed.
```

Comment:
I am not sure whether the new description is better. I think that the problem is in `Updates each specified package to the latest available version that provides`.  
What about: 
`Upgrades each specified package to the to the first available version that provides a bugfix, enhancement or a fix for security issue (security).`

It means that it does not have to upgrade to version mentioned in advisory. Any version after version in advisory  provides advisory.  This situation often happen in Fedora, but it can happen even  for RHEL. I think that `Newer updates not mentioned in any advisories are ignored.` is misleading and conflicting with the first sentence.

Reply:
> Any version after version in advisory provides advisory.

Oh, I see. I was not thinking about it in this cummulative manner. Thanks for clarification. I will try to reformulate it.